mne/gui/_coreg.py

```diff
@@ -491,26 +538,21 @@ def _lock_fids_changed(self, change=None):
             self._scale_mode_changed()
             self._display_message()
             self._update_distance_estimation()
+            self._forward_widget_command('save_mri_fids', 'set_enabled', True)
```

Comment:
I think we can add `'save_mri_fids'` to the `locked_widgets` list instead of calling `_forward_widget_command` directly.

Reply:
Done. 

Also see #10262 where I'm planning to address the "locking issue" more thoroughly.